src/Symfony/Component/Security/Core/Security.php

```diff
@@ -12,27 +12,34 @@
 namespace Symfony\Component\Security\Core;
 
 use Psr\Container\ContainerInterface;
+use Symfony\Bundle\SecurityBundle\Security\Security as NewSecurityHelper;
 use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
 use Symfony\Component\Security\Core\User\UserInterface;
 
 /**
  * Helper class for commonly-needed security tasks.
  *
- * @final
+ * @deprecated since Symfony 6.2, use {@see NewSecurityHelper} instead
```

Comment:
Aliased use statements won't be resolved in the messages of the DebugClassLoader. So you should rather use a FQCN here.

Reply:
Thanks